ios/FluentUI.Demo/FluentUI.Demo/Resources/Assets.xcassets/LeftNav/ic_fluent_settings_24_regular.imageset/Contents.json

```diff
@@ -2,23 +2,15 @@
   "images" : [
     {
       "filename" : "ic_fluent_settings_24_regular.pdf",
-      "idiom" : "universal",
-      "scale" : "1x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "2x"
-    },
-    {
-      "idiom" : "universal",
-      "scale" : "3x"
+      "idiom" : "universal"
     }
   ],
   "info" : {
     "author" : "xcode",
     "version" : 1
   },
   "properties" : {
+    "preserves-vector-representation" : true,
```

Comment:
I forget... is this the latest guidance to use this?

Reply:
I believe so? My understanding is that a single vector is quite a bit smaller than multiple bitmaps, and that the performance impact is now negligible. But I can ask around to verify.

Reply:
https://microsoft.github.io/apple-ux-guide/iOSImageFileFormat.html#app-size-impact

Reply:
Huh! Okay, reverted back to bitmap only. Thanks for catching that!